ionoscloud/data_source_dataplatform_cluster.go

```diff
@@ -269,16 +309,21 @@ func setAdditionalDataplatformClusterData(ctx context.Context, d *schema.Resourc
 
 	/* get from api and set in schema the kubeconfig*/
 	if cluster.Id != nil {
-		kubeConfig, _, err := client.GetClusterKubeConfig(ctx, *cluster.Id)
+		kubeConfigMap, _, err := client.GetClusterKubeConfig(ctx, *cluster.Id)
 		if err != nil {
 			return fmt.Errorf("an error occurred while fetching the kubernetes config for cluster with ID %s: %w", *cluster.Id, err)
 		}
 
-		if err := d.Set("kube_config", kubeConfig); err != nil {
+		kubeConfig, err := json.Marshal(kubeConfigMap)
+		kubeConfigString := string(kubeConfig)
```

Comment:
first check for error I think

Reply:
@cristiGuranIonos done in cdd9b58070e612723abae0806410ff7d12151ea5